app.py

```diff
@@ -437,6 +444,7 @@ def parse():
         'code': code,
         'server_error': response.get('Error'),
         'exception': get_class_name(exception),
+        'alternative_error_message': alt_err_msgs,
```

Comment:
What happens if `alt_err_msgs` is empty? It is unclear to me what value this variable can contain.

Reply:
As you can see in line 380, `app.py`:
`# Check if alternative error messages are available in this parse`
`alt_err_msgs = alternative_error_messages.is_available()`

Always a boolean value is returned and we have to log it to analyze it later.